src/pages/_app.tsx

```diff
@@ -25,6 +26,7 @@ function MyApp({ Component, pageProps }: AppProps) {
       <ClerkProvider clerkJSVariant="headless" {...pageProps}>
         <QueryClientProvider client={queryClient}>
           <main className={`${inter.className}`}>
+            <NavBar />
```

Comment:
This shows up on the login and register screens as well, which is unneeded. Take a look at https://nextjs.org/docs/pages/building-your-application/routing/pages-and-layouts#per-page-layouts for how to do this